lms/djangoapps/courseware/access.py

```diff
@@ -439,49 +435,6 @@ def check_for_staff():
     return _dispatch(checkers, action, user, descriptor)
 
 
-NONREGISTERED_CATEGORY_WHITELIST = [
```

Comment:
I think we want this,
but maybe we can move it to `openedx/stanford/lms/envs/common.py`